src/components/timeline/TimelineItems.css

```diff
@@ -1,265 +1,288 @@
 div.timeline-container {
-    padding: 0;
```

Comment:
Could you perhaps explain what you changed in this css file? Sorry I'm having some trouble going through it.

Reply:
Yeah, the whole thing auto-formatted when I saved the changes.

 - I removed the padding from h4.cityState
 - I added h3.card-desc

Reply:
Thanks!